README.md

```diff
@@ -1,4 +1,4 @@
 ## This is my Hello Microverse' project
 
 
-This is my first project at microverse
+This is my first project day 2

```

Comment:
- [ ] As requested by the previous reviewer please consider using a well-documented and organized README file it will make your project looks more professional and helps readers understand the purpose of this repo, kindly use this [README template](https://github.com/microverseinc/readme-template) provided by Microverse and try to customize so it will be more specific to this project.